.github/ISSUE_TEMPLATE/--question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "❓ Question"
+about: Please use https://discuss.lightbend.com/c/akka/streams-and-alpakka for questions
+
+---
+
+If you have a [Lightbend Subscription](https://www.lightbend.com/lightbend-platform-subscription), please reach out via the [Lightbend Portal](https://portal.lightbend.com/).
+
+Otherwise, please use the [Lightbend discuss forum](https://discuss.lightbend.com/c/akka/streams-and-alpakka) for questions instead of posting them to the issue tracker.
```

Comment:
Should this also _warns_ users that the issue will probably be closed without further action?